Source/DafnyCore/AST/DafnyAst.cs

```diff
@@ -41,6 +41,10 @@ public abstract class INode {
     /// </summary>
     public abstract IEnumerable<INode> Children { get; }
 
+    public IEnumerable<INode> DeepChildren() {
```

Comment:
This is extremely subjective, but I'm used to calling a property like this `Descendants`

Reply:
Yeah, I like that name better, too. :)